website/static/js/myProjects.js

```diff
@@ -634,7 +634,8 @@ var MyProjects = {
                             'You have not created any projects yet.');
                     } else if (lastcrumb.data.nodeType === 'registrations'){
                         template = m('.db-non-load-template.m-md.p-md.osf-box',
-                            'You have not made any registrations yet.');
+                            'You have not made any registrations yet. Go to ',
+                            m('a', {href: 'help.osf.io/#registrations'}, 'Getting Started'), ' to learn how registrations work.' );
```

Comment:
Missing `http://` on this link. This tries to take you to `http://localhost:5000/myprojects/help.osf.io/#registrations` which is not a thing.